src/shared/hooks/index.tsx

```
@@ -1,7 +1,7 @@
-import { PluginSettings } from "@/constants/settings";
+import { PluginSettings } from "@/shared/constants/settings";
```

Biome (1.9.4)

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)